### PR TITLE
[hotfix][k8s] Fix k8s ha service doc.

### PR DESCRIPTION
**Pull request by wanglijie95**
## What is the purpose of the change

Fix the error doc in `KubernetesHaServices`.

### CODE REVIEW

**Comment by wanglijie95** [comment]
cc @wangyang0918

File: flink-kubernetes/src/main/java/org/apache/flink/kubernetes/highavailability/KubernetesHaServices.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
 import static org.apache.flink.util.Preconditions.checkNotNull;
 
 /**
- * An implementation of the {@link AbstractHaServices} using Apache ZooKeeper.
+ * An implementation of the {@link AbstractHaServices} based on kubernetes API.
```
Comment (wangyang0918) [inline flink-kubernetes/src/main/java/org/apache/flink/kubernetes/highavailability/KubernetesHaServices.java:45]:
```suggestion
 * An implementation of the {@link AbstractHaServices} using Kubernetes.
```

**Review by wangyang0918 — COMMENTED** [review]
Thanks for creating this PR. LGTM. I just java a minor suggestion.

File: flink-kubernetes/src/main/java/org/apache/flink/kubernetes/highavailability/KubernetesHaServices.java (hunk 1 above)
Comment (wanglijie95) [reply flink-kubernetes/src/main/java/org/apache/flink/kubernetes/highavailability/KubernetesHaServices.java]: Thanks for review. I have modified according to your suggestion.

**Review by wangyang0918 — APPROVED** [review]

**Comment by wangyang0918** [comment]
cc @xintongsong could you help with merging this fix?

**Review by xintongsong — APPROVED** [review]
LGTM. The test failure is un-related. Merging this.

**Pull request closed without merging**